fuel-tests/tests/coin.rs

```diff
@@ -222,12 +229,13 @@ async fn only_unspent_coins() {
         .all(|c| c.status == SchemeCoinStatus::Unspent));
 }
 
+#[rstest]
 #[tokio::test]
-async fn coins_to_spend() {
-    let owner = Address::default();
-    let asset_id_a = AssetId::new([1u8; 32]);
-    let asset_id_b = AssetId::new([2u8; 32]);
-
+async fn coins_to_spend(
```

Comment:
It will add a headache to my pull request=D Because I split it [into small subtests and added support for messages](https://github.com/FuelLabs/fuel-core/blob/feature/coins-to-spent-messages/fuel-tests/tests/resource.rs#L29)

Reply:
If this was majorly redone, we should leave the coins_to_spend test as is in this PR then.

Reply:
@xgreenx would you prefer if I keep this test alone then for this PR?

Reply:
I prefer to avoid conflicts=D But as you wish